### DISCUSSION TITLE
Can someone provide a detailed guide to get root access please?

### DISCUSSION THREAD

**Discussion by Fallenmoon8080** [3kh0/ext-remover#172] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can someone provide a detailed guide for root access. I am on chomeOS Version 102.0.5005.75 (Official Build) (64-bit). Crosh is unblocked. I cannot download extensions or add devmode or alt accounts. Thank you. Please do not redirect me to discord or bugs.chromium.org. Thank you

**Comment by ghost** [comment, marked as answer]
One thing is for sure: You can't downgrade or unenroll
